flowkit/arguments/arguments.go

```diff
@@ -145,14 +147,21 @@ func ParseWithoutType(args []string, code []byte, fileName string) (scriptArgs [
 			break
 		}
 
-		var value, err = runtime.ParseLiteral(argumentString, semaType, inter)
-		if err != nil {
-			return nil, fmt.Errorf(
-				"argument `%s` is not expected type `%s`",
-				parameterList[index].Identifier,
-				semaType.QualifiedString(),
-			)
+		var value cadence.Value
+
+		if isOptional && argumentString == "nil" {
+			value = cadence.NewOptional(nil)
```

Comment:
The value must be wrapped in as many additional optionals as the type indicates. For example:
- `T?`: `cadence.NewOptional(nil)`
- `T??`: `cadence.NewOptional(cadence.NewOptional(nil))`

Reply:
thanks I will fix asap.

Reply:
fixed with correct parseLiteral usage.